packaging/Shell/install-fedora-deps-dev.sh

```diff
@@ -47,43 +47,43 @@ dnf -y debuginfo-install \
 # CIMpp and VILLASnode are installed here
 LD_LIBRARY_PATH="/usr/local/lib64:${LD_LIBRARY_PATH}"
 
-# minimal VILLASnode dependencies
+# Minimal VILLASnode dependencies
 dnf -y install \
     openssl-devel \
     libuuid-devel \
     libcurl-devel \
     jansson-devel \
     libwebsockets-devel
 
-# optional VILLASnode dependencies
+# Optional VILLASnode dependencies
 dnf -y install \
     mosquitto-devel \
-	libconfig-devel \
+    libconfig-devel \
     libnl3-devel
 
-pip3 install -U setuptools
-pip3 install -U wheel
-pip3 install -r requirements.txt
+pip3 install --upgrade setuptools
+pip3 install --upgrade wheel
+pip3 install .[test,docs]
```

Comment:
Here looks like we are doing a different thing, not only installing dependencies but also the package in the currently active folder (dpsim). Is this the intended behaviour?

Reply:
In the dev container, the dpsim package should not be installed... only the dependencies.